src/Illuminate/Cache/Repository.php

```diff
@@ -185,11 +185,33 @@ protected function handleManyResult($keys, $key, $value)
      */
     public function pull($key, $default = null)
     {
+        if (is_array($key)) {
+            return $this->pullMany($key, $default);
+        }
+        
```

Comment:
This is still a breaking change. The array was handled by the `get` method below. Now it's not passed down anymore.

Honestly, this change has been attempted lots of times in the past and always got turned down.

Reply:
So, why the `forget` method in `src/Illuminate/Cache/Repository.php` only accept the param `$key` as a string ,but the `pull` method, which use the `forget` method , can pass the `$key` as a array ?

Reply:
I have removed this breaking change section and written it another way.

Reply:
Maybe it is an annotation issue instead of a code issue, and I have read the Laravel documentation but no array keys parameter using for the ```Cache::pull``` method.
We only should modify the parameter `$key` annotation for the ```pull``` method
It is my bug!

Let me know what your thought is.